components/combobox/index.css

```diff
@@ -56,6 +48,7 @@
 	--spectrum-combobox-border-color-focus: var(--spectrum-gray-800);
 	--spectrum-combobox-border-color-focus-hover: var(--spectrum-gray-900);
 	--spectrum-combobox-border-color-key-focus: var(--spectrum-gray-800);
+	--spectrum-combobox-border-color-disabled: var(--spectrum-disabled-border-color);
```

Comment:
This was undefined post-merge and needed to be brought over from the spectrum-two theme.

Reply:
If this wasn't coming from main - I'd rather we not try to add it in this PR now. We know there are lint issues on spectrum-two but that's not really a problem until we're ready to release.